app/src/ereferrals/java/org/eyeseetea/malariacare/domain/entity/UIDGenerator.java

```diff
@@ -0,0 +1,16 @@
+package org.eyeseetea.malariacare.domain.entity;
+
+import java.util.Date;
+
+public class UIDGenerator {
```

Comment:
we should create unit tests that verifying business rules with uid according to the client